ts/Data/DataTable.ts

```diff
@@ -1601,8 +1695,27 @@ class DataTable implements DataEvent.Emitter {
         });
     }
 
+    // The key ID column must always be the last column
+    private moveKeyIdColumnToLast(columns: Record<string, DataTable.Column>, id: string): void {
+        const rowKeyColumn = columns[id];
+        delete columns[id];
+        columns[id] = rowKeyColumn;
+    }
+
+    // The key ID column must removed in some methods
+    // (API backwards compatibility)
+    private removeKeyIdColumn(columnNamesOrAliases: Array<string>): void {
```

Comment:
Can we maybe settle on a name similar to `filterRowKeysId`? I was confusing it with deleteColumn in other code lines.